### PR TITLE
[Merged by Bors] - DEVOPS-605 Labeler auth token change

### PR DESCRIPTION
**Pull request by jorge-cr-13**
Changing the authentication token for the labeler CI to GITHUB_TOKEN.

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -36,13 +36,13 @@ jobs:
         uses: buildsville/add-remove-label@6008d7bd99d3baeb7c04033584e68f8ec80b198b # v1
         if: contains(github.event.pull_request.labels.*.name, 'WIP ⏳')
         with:
-          token: ${{secrets.LABEL_MODIFIER_TOKEN_GITHUB}}
+          token: ${{secrets.GITHUB_TOKEN}}
           label: "WIP ⏳"
           type: remove
-
+
```
Comment (Robert-Steiner) [inline .github/workflows/labeler.yml:42]:
```suggestion

```
some spaces have crept in :)

**Review by Robert-Steiner — APPROVED** [review]
Nice 👍

As a small addition (doesn't have to be in this pr) we can now also restrict the permissions of the `GITHUB_TOKEN` via the [permissions](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#permissions) keyword. The action we use uses the pull request api therefore we will probably need to set the `pull-requests` option to `write`:

```yaml
permissions:
  pull-requests: write  
```

[overview of the permissions](https://docs.github.com/en/rest/reference/permissions-required-for-github-apps)

**Comment by jorge-cr-13** [comment]
> Nice 👍
> 
> As a small addition (doesn't have to be in this pr) we can now also restrict the permissions of the `GITHUB_TOKEN` via the [permissions](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#permissions) keyword. The action we use uses the pull request api therefore we will probably need to set the `pull-requests` option to `write`:
> 
> ```yaml
> permissions:
>   pull-requests: write  
> ```
> 
> [overview of the permissions](https://docs.github.com/en/rest/reference/permissions-required-for-github-apps)

Cool I think it is an amazing addition. I'm just going to test it and immediately add it to this PR. So all the changes go on the same.

**Comment by jorge-cr-13** [comment]
> > Nice 👍
> > As a small addition (doesn't have to be in this pr) we can now also restrict the permissions of the `GITHUB_TOKEN` via the [permissions](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#permissions) keyword. The action we use uses the pull request api therefore we will probably need to set the `pull-requests` option to `write`:
> > ```yaml
> > permissions:
> >   pull-requests: write  
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > [overview of the permissions](https://docs.github.com/en/rest/reference/permissions-required-for-github-apps)
> 
> Cool I think it is an amazing addition. I'm just going to test it and immediately add it to this PR. So all the changes go on the same.

Ok, I tested it and yes with the PR write permissions is enough. Thanks again for the note!

**Comment by jorge-cr-13** [comment]
bors merge

**Pull request closed without merging**